docs/build/mapping.md

```diff
@@ -144,6 +144,17 @@ An AvalancheEvent encapsulates Event data and Transaction/Block information corr
 
 A CosmosEvent/TerraEvent encapsulates Event data and TxLog corresponding to the event. It also contains CosmosMessage/TerraMessage data of the message connected to the event. Also, it includes the CosmosBlock/TerraBlock and CosmosTransaction/TerraTransaction data of the block and transaction from which the event was emitted.
 
+::: info Note
+From `@subql/types` version `X.X.X` onwards `SubstrateEvent` is now generic. This can give you better typesafety when developing your project.
```

Comment:
@stwiname Maybe, use such sentence (here and below):

This can provide you with higher type safety when developing your project.